### PR TITLE
NMS-15290: Add outgoing connection to TS DB doc

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `deployment/time-series-storage/timeseries/hosted-tss.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://opennms.atlassian.net/browse/NMS-15290

### CODE REVIEW

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
+Follow these steps to set up the Time Series DB in your environment:
 
-. Update OpenNMS Cloud Services Connector files.
-. Configure plugin settings.
+. Update your OpenNMS Cloud Services Connector files.
+. Configure Cloud Services settings.
 . Obtain an activation key for the Cloud Services Connector.
-. Enable cloud services connectivity.
+. Enable Cloud Services connectivity.
+. Allow {page-component-title} to make outgoing connections to the following gateways:
```
Comment (mmahacek) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:26]: This should be checked first, as you can't enable the cloud services unless the firewall is allowing access to these hosts.
We may also need to provide IP addresses, as firewalls typically base their rules on IP and not the DNS name.

Comment (emarsh1995) [reply docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:26]: Would it be sufficient to include an admonition for the firewall check in the step's section, or are there specific steps we should write out to guide the user through checking this?

File: docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
+** TCP `auth.access.onms-dp-prod.production.prod.dataservice.opennms.com:443`
+** TCP `gateway.onms-dp-prod.production.prod.dataservice.opennms.com:443`
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc:28]:
```suggestion
** TCP `gateway.onms-dp-prod.production.prod.dataservice.opennms.com:443`

NOTE: You should ensure that your firewall allows access to the gateways above.
If they are blocked, you cannot enable the Cloud Services.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Review by mmahacek — APPROVED** [review]

**Pull request merged**